TokenAcquirerTests/TokenAcquirer.cs

```diff
@@ -103,7 +103,7 @@ public async Task AcquireTokenWithPop_ClientCredentialsAsync()
             var cert = options.ClientCredentials.First().Certificate;
 
             // Get the token acquisition service
-            ITokenAcquirer tokenAcquirer = serviceProvider.GetRequiredService<ITokenAcquirer>();
+            ITokenAcquirer tokenAcquirer = tokenAcquirerFactory.GetTokenAcquirer(string.Empty);
```

Comment:
do we need to support the case with no authentication scheme?

Reply:
I renamed the parameter to optionName, and set it to string.Empty by default.